### PR TITLE
Config option to control output of desc.txt and narrator.txt

### PR DESCRIPTION
**Pull request by SenorSmartyPants**
Defaults to outputting the files to maintain current behavior

### CODE REVIEW

**Comment by SenorSmartyPants** [comment]
Rebased to fix merge conflict.

Addresses #1

File: readme.md [hunk 1]
```diff
@@ -53,6 +53,8 @@ This Beets plugin solves both problems.
      source_weight: 0.0 # disable the source_weight penalty
      fetch_art: true # whether to retrieve cover art
      include_narrator_in_artists: true # include author and narrator in artist tag. Or just author
+     desc_txt': true # output desc.txt
+     reader_txt': true # output reader.txt
```
Comment (Neurrone) [inline readme.md:57]: 1. There's an extra `'` before the `:`, is that a typo?
2. What do you think about renaming the new options  to `write_description_file` and `write_reader_file`?

**Review by Neurrone — CHANGES_REQUESTED** [review]
Thanks, looks good.

File: readme.md (hunk 1 above)
Comment (SenorSmartyPants) [reply readme.md]: Yep, typo. Updated option names.

**Review by Neurrone — APPROVED** [review]

**Pull request merged**